docker-compose.yml

```diff
@@ -1,11 +1,28 @@
-version: '3.7'
+version: "3.7"
 
 services:
+  influxdb:
+    image: library/influxdb:1.8
+    container_name: influxdb
+    ports:
+      - "8086:8086"
+    networks:
+      - tsm
+
   tsm:
     image: tsm
```

Comment:
can we switch to `build: .` from `image` ? this would build fresh image for every single start - it prevents from mistake of not rebuilding after a fix